README.md

```diff
@@ -9,3 +9,16 @@
 `rails db:migrate`
 
 `rails hydra:server`
+
+# Quickstart development with Docker
+Docker provides an alternative way of setting up an Avalon Media System Development Environment in minutes without installing any dependencies beside Docker itself. It should be noted that the docker-compose.yml provided here is for development only and will be updated continually.
+* Install [Docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/)
+* ```git clone https://github.com/avalonmediasystem/tbd```
```

Comment:
Please change this to samvera-labs/avalon-bundle in anticipation of transfer to labs.

Reply:
✅